themes/default/index.html

```diff
@@ -226,9 +246,17 @@
                 <!-- Body content -->
                 <tr>
                   <td class="content-cell">
-                    <h1><%- greeting %> <%- name %>,</h1>
-                    <p><%- intro %></p>
-                    
+                    <% if (locals.title) { %>
```

Comment:
This is nice, but I think we can improve this by only exposing the `title` variable to the templates, and implementing logic within `parseParams()` (e.g. [here](https://github.com/AlecRust/mailgen/blob/b5a4e0245f091d6b24b1002a33be14df569927c1/index.js#L102)) to set the title variable to "{greeting} {name}," if `body.title` is not specified.

What do you think?